fix bug 4th try

Run withastro/action@v5
Run len=`echo $INPUT_PM | wc -c`
Run pnpm/action-setup@41ff72655975bd51cab0327fa583b6e92b6d3061
Running self-installer...
Installation Completed!
Run actions/setup-node@2028fbc5c25fe9cf00d9f06a71cc4710d4507903
Found in cache @ /opt/hostedtoolcache/node/24.15.0/x64
(node:2273) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
Environment details
/home/runner/setup-pnpm/node_modules/.bin/pnpm store path --silent
/home/runner/setup-pnpm/node_modules/.bin/store/v3
pnpm cache is not found
Run if [ "$PACKAGE_MANAGER" = "deno" ]; then
Progress: resolved 1, reused 0, downloaded 0, added 0
Progress: resolved 71, reused 0, downloaded 56, added 0
Progress: resolved 243, reused 0, downloaded 189, added 0
Progress: resolved 487, reused 0, downloaded 353, added 0
Packages: +362
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Progress: resolved 487, reused 0, downloaded 374, added 362, done
.../esbuild@0.27.7/node_modules/esbuild postinstall$ node install.js
.../esbuild@0.25.12/node_modules/esbuild postinstall$ node install.js
.../sharp@0.34.5/node_modules/sharp install$ node install/check.js || npm run build
.../esbuild@0.27.7/node_modules/esbuild postinstall: Done
.../esbuild@0.25.12/node_modules/esbuild postinstall: Done
.../sharp@0.34.5/node_modules/sharp install: Done
dependencies:
+ @astrojs/mdx 4.3.14 (5.0.6 is available)
+ @astrojs/rss 4.0.18
+ @astrojs/sitemap 3.7.2
+ @fontsource-variable/inter 5.2.8
+ @fontsource-variable/newsreader 5.2.10
+ @tailwindcss/vite 4.3.0
+ astro 5.18.1 (6.3.6 is available)
+ marked 17.0.6 (18.0.4 is available)
+ sharp 0.34.5
+ tailwindcss 4.3.0
devDependencies: skipped because NODE_ENV is set to production
Done in 4.6s using pnpm v9.15.9
Run actions/cache/restore@9255dc7a253b0ccc959486e2bca901246202afeb
Cache not found for input keys: astro-cache-Linux-6a6efd05817d541c188d369648df0bf0740b850a, astro-cache-Linux-
Run pnpm run build
> patrick-omara-website@1.0.0 build /home/runner/work/omaradigital/omaradigital
> NODE_ENV=production astro build
01:06:02 [content] Syncing content
01:06:02 [content] Synced content
01:06:02 [types] Generated 707ms
01:06:02 [build] output: "static"
01:06:02 [build] mode: "static"
01:06:02 [build] directory: /home/runner/work/omaradigital/omaradigital/dist/
01:06:02 [build] Collecting build info...
01:06:02 [build] ✓ Completed in 747ms.
01:06:02 [build] Building static entrypoints...
01:06:03 [ERROR] [vite] ✗ Build failed in 754ms
[@tailwindcss/vite:generate:build] Can't resolve '@tailwindcss/typography' in '/home/runner/work/omaradigital/omaradigital/src/styles'
file: /home/runner/work/omaradigital/omaradigital/src/styles/global.css
  Stack trace:
    at finishWithoutResolve (/home/runner/work/omaradigital/omaradigital/node_modules/.pnpm/enhanced-resolve@5.21.6/node_modules/enhanced-resolve/lib/Resolver.js:942:18)
    at /home/runner/work/omaradigital/omaradigital/node_modules/.pnpm/enhanced-resolve@5.21.6/node_modules/enhanced-resolve/lib/Resolver.js:1111:5
    at /home/runner/work/omaradigital/omaradigital/node_modules/.pnpm/enhanced-resolve@5.21.6/node_modules/enhanced-resolve/lib/Resolver.js:1111:5
    at /home/runner/work/omaradigital/omaradigital/node_modules/.pnpm/enhanced-resolve@5.21.6/node_modules/enhanced-resolve/lib/DescriptionFilePlugin.js:94:43
    at eval (eval at create (/home/runner/work/omaradigital/omaradigital/node_modules/.pnpm/tapable@2.3.3/node_modules/tapable/lib/HookCodeFactory.js:31:10), <anonymous>:15:1)
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.

--- a/package.json
+++ b/package.json
@@ -15,6 +15,7 @@
         "@astrojs/sitemap": "^3.6.0",
         "@fontsource-variable/inter": "^5.2.8",
         "@fontsource-variable/newsreader": "^5.2.10",
+        "@tailwindcss/typography": "^0.5.19",
         "@tailwindcss/vite": "^4.1.17",
         "astro": "^5.15.9",
         "marked": "^17.0.0",
@@ -25,7 +26,6 @@
         "onlyBuiltDependencies": ["esbuild", "sharp"]
     },
     "devDependencies": {
-        "@tailwindcss/typography": "^0.5.19",
         "prettier": "^3.6.2",
         "prettier-plugin-astro": "^0.14.1",
         "prettier-plugin-tailwindcss": "^0.7.1",

